src/Components/InputField/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { schema } from "/home/mergestack/Desktop/Fakhar Training/React/Form + Yup to Ts/my-redux-hook-form-ts-app/src/Components/InputField/schema";
+
+interface Props {
+  addTodo: (value: any) => void;
+}
+
```

Comment:
use proper interfaces and follow convention like IInputField which represents interface of inputfield